crates/router/src/connector/netcetera.rs

```diff
@@ -365,20 +378,21 @@ impl
         req: &types::authentication::ConnectorAuthenticationRouterData,
         _connectors: &settings::Connectors,
     ) -> CustomResult<RequestContent, errors::ConnectorError> {
-        let connector_router_data = netcetera::NetceteraRouterData::try_from((
-            &self.get_currency_unit(),
+        let currency =
             req.request
                 .currency
                 .ok_or(errors::ConnectorError::MissingRequiredField {
                     field_name: "currency",
-                })?,
+                })?;
+        let amount =
             req.request
                 .amount
```

Comment:
The type of this `amount` field must be changed from `Option<i64>` to `Option<MinorUnit>`.

All amount types in hyperswitch core must be `MinorUnit` instead of `i64`.

Reply:
Hi @hrithikesh026, i am a bit confused. do you want me to change the type from i64 to MinorUnit in [hyperswitch_domain_models](https://github.com/juspay/hyperswitch/blob/342529e0565baaa02f33266c3be620a9561048c8/crates/hyperswitch_domain_models/src/router_request_types/authentication.rs#L83) crate. If it thats the case i have to change its references as well. Please clarify

Reply:
Yes. `amount` field in `struct ConnectorAuthenticationRequestData` must be of type `Option<MinorUnit>`.

Reply:
This change is being marked as resolved due to the higher effort it would require